src/styles/stars.scss

```
@@ -0,0 +1,29 @@
+.stars {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 24px;
+
+  &__reviews {
+    font-size: 10px;
+    color: var(--main-letters-color);
+    display: flex;
+    justify-content: space-between;
+    margin-left: 17px;
+    line-height: 14px;
+  }
+
+  &__star {
+    background-position: center;
+    background-repeat: no-repeat;
+    background-image: url(../images/star.svg);
+    height: 16px;
+    width: 16px;
+  }
+}
+
+@for $starcount from 0 through 5 {
+  .stars--#{$starcount} :nth-child(-n+#{$starcount}) {
+    background-image: url(../images/star-active.svg);
+  }
+}
```

```suggestion
@for $starcount from 0 through 5 {
  &--#{$starcount} :nth-child(-n+#{$starcount}) {
    background-image: url(../images/star-active.svg);
  }
}
}

```

It should be nested into `.stars` block